rsocket-core/src/main/java/io/rsocket/core/RSocketRequester.java

```diff
@@ -560,46 +560,58 @@ private void handleStreamZero(FrameType type, ByteBuf frame) {
 
   private void handleFrame(int streamId, FrameType type, ByteBuf frame) {
     Subscriber<Payload> receiver = receivers.get(streamId);
-    if (receiver == null) {
```

Comment:
this statement is basically a bug which prevents handling the `requestN` frames when the receiver is completed

Reply:
cc/ @simonbasle @rstoyanchev